tests/Functional/SetAclCommandTest.php

```diff
@@ -41,7 +41,9 @@ public function testSetAclRole()
         $role = 'ROLE_ADMIN';
 
         $application = $this->getApplication();
-        $application->add(new SetAclCommand($application->getKernel()->getContainer()->get('security.acl.provider')));
+        $application->add(new SetAclCommand(
+            self::$container->get('security.acl.provider'),
+            self::$container->get('security.acl.permission.map')));
```

Comment:
`static::getContainer()` should be preferred as of 5.3. Moreover, calling `add()` should not be necessary in a functional test, I'd expect the command to be already registered, I suppose we can just remove the call here.